README.md

```diff
@@ -17,158 +17,30 @@ Add this line to your application's Gemfile:
 gem 'onfido', '~> 2.0.0'
 ```
 
-## Configuration
+## Getting started
 
-There are 5 configuration options:
+Configure with your API token and region:
 
 ```ruby
-Onfido.configure do |config|
-  config.api_key = '<YOUR_API_KEY>'
-  config.logger = Logger.new(STDOUT)
-  config.open_timeout = 30
-  config.read_timeout = 80
-  config.region = nil
-end
-```
-
-### Regions
-
-The gem will use the default region if no region is specified.
-
-To specify the US region do:
-`config.region = :us`
-
-To specify the CA region do:
-`config.region = :ca`
-
-See https://documentation.onfido.com/#regions for supported regions.
-
-## Usage
-
-You can make API calls by using an instance of the `API` class:
-
-```ruby
-onfido = Onfido::API.new
-```
-
-You can also set an API key as follows, instead of in the initializer configuration:
-
-```ruby
-onfido = Onfido::API.new(api_key: '<YOUR_API_KEY>')
+onfido = Onfido::API.new(
+  api_key: ENV['ONFIDO_API_KEY'],
+  # Supports :eu, :us and :ca. Previously defaulted to :eu.
+  region: :eu
+)
 ```
 
-### Resources
```

Comment:
Removing all duplicated documentation in favour of pointing to documentation.onfido.com - the same as we've done for other client libraries.